### DISCUSSION TITLE
Jetbrains IDE scaling issues

### DISCUSSION THREAD

**Discussion by jiangdengke** [prasanthrangan/hyprdots#2084] [Q&A]
Hello everyone,I may need your help,The original post is here, https://github.com/prasanthrangan/hyprdots/issues/2074   ，I first encountered the problem that jetbrains IDE is too large,And then make idea run on wayland,But jetbrains IDE can't use the input method The problem has not been solved,So I can only come back and run with xwayland,But although the kind people have given me the answer ，Because I'm a novice, I still haven't solved it.
```bash 
xwayland {
    force_zero_scaling = false
}
I tried adding in .config/hypr/monitors.conf but it didn't work, if you helped me I would be very grateful

**Comment by kRHYME7** [comment]
set it to true

**Reply by jiangdengke** [reply]
Hello, I just set it to true, but the interface is still very large, should it have other settings to make the interface look normal

**Reply by jiangdengke** [reply]
Just set up the zoom in the IDE and it's OK.,This should be the bad step.,Thank you.

**Reply by kRHYME7** [reply]
> Hello, I just set it to true, but the interface is still very large, should it have other settings to make the interface look normal

env = GDK_SCALE,1
(note that env variables needs reboot to take effect)

but this will affect all Xwayland windows. If not working your IDE might be zoomed.
